src/main/resources/assets/js/app/wizard/UpdatePersistedContentRoutine.ts

```diff
@@ -91,7 +88,13 @@ export class UpdatePersistedContentRoutine
     }
 
     private hasContentChanged(persisted: Content, viewed: Content): boolean {
-        return !persisted.getContentData().equals(viewed.getContentData());
+        return this.workflowState === WorkflowState.READY ||
+               !persisted.dataEquals(viewed.getContentData()) ||
+               !persisted.getOwner().equals(viewed.getOwner()) ||
+               !persisted.getPermissions().equals(viewed.getPermissions()) ||
+               persisted.getLanguage() !== viewed.getLanguage() ||
+               persisted.getPublishFromTime() !== viewed.getPublishFromTime() ||
+               persisted.getPublishToTime() !== viewed.getPublishToTime();
     }
```

Comment:
Shouldn't the conditions here compare all the same properties that are set in `produceUpdateContentRequest`?

```
            .setRequireValid(this.requireValid)
            .setContentName(viewedContent.getName())
            .setDisplayName(viewedContent.getDisplayName())
            .setData(viewedContent.getContentData())
            .setExtraData(viewedContent.getAllExtraData())
            .setOwner(viewedContent.getOwner())
            .setLanguage(viewedContent.getLanguage())
            .setPublishFrom(viewedContent.getPublishFromTime())
            .setPublishTo(viewedContent.getPublishToTime())
            .setPermissions(viewedContent.getPermissions())
            .setInheritPermissions(viewedContent.isInheritPermissionsEnabled())
            .setOverwritePermissions(viewedContent.isOverwritePermissionsEnabled())
            .setWorkflow(workflow);
```

Reply:
True, updated the PR